applications/nrf_desktop/src/modules/dfu.c

```diff
@@ -52,29 +51,48 @@ LOG_MODULE_REGISTER(MODULE, CONFIG_DESKTOP_CONFIG_CHANNEL_DFU_LOG_LEVEL);
 #define SYNC_BUFFER_SIZE (CONFIG_DESKTOP_CONFIG_CHANNEL_DFU_SYNC_BUFFER_SIZE * sizeof(uint32_t)) /* bytes */
 
 #if CONFIG_SECURE_BOOT
- #include <fw_info.h>
- #define IMAGE0_ID		PM_S0_IMAGE_ID
- #define IMAGE0_ADDRESS		PM_S0_IMAGE_ADDRESS
- #define IMAGE0_SIZE		PM_S0_IMAGE_SIZE
- #define IMAGE1_ID		PM_S1_IMAGE_ID
- #define IMAGE1_ADDRESS		PM_S1_IMAGE_ADDRESS
- #define IMAGE1_SIZE		PM_S1_IMAGE_SIZE
- #define BOOTLOADER_NAME	"B0"
+	BUILD_ASSERT(IS_ENABLED(CONFIG_PARTITION_MANAGER_ENABLED),
+		     "B0 bootloader supported only with Partition Manager");
+	#include <pm_config.h>
+	#include <fw_info.h>
+	#define BOOTLOADER_NAME	"B0"
+	#if PM_ADDRESS == PM_S0_IMAGE_ADDRESS
+		#define DFU_SLOT_ID		PM_S1_IMAGE_ID
+	#elif PM_ADDRESS == PM_S1_IMAGE_ADDRESS
+		#define DFU_SLOT_ID		PM_S0_IMAGE_ID
+	#else
+		#error Missing partition definitions.
+	#endif
```

Comment:
I think we don't indent content of the `#if` blocks

Reply:
Personally, I think it's far more readable with the indent. The approach was accepted e.g. in `ncs/nrf/applications/machine_learning/src/modules/sensor_sim_ctrl.c`. If you insist I can remove the indents (unfortunately compliance does not allow for indents using spaces).

Reply:
discussed offline